backend/middleware/isAdmin.js

```diff
@@ -0,0 +1,15 @@
+
+
+export const isAdmin = (req, res, next) => {
+    try {
+
+        const role = req.user.role
```

Comment:
Like I mentioned in another comment, it's securing the authorization level of a particular user, even though the token cannot change the data due to the nature of JWT.